packages/grid/_modules_/grid/components/styled-wrappers/GridRootStyles.ts

```diff
@@ -134,6 +134,10 @@ export const useStyles = makeStyles(
           },
           '&.Mui-resizing': {
             color: theme.palette.text.primary,
+            // Reset on touch devices, it doesn't add specificity
+            '@media (hover: none)': {
+              color: borderColor,
+            },
```

Comment:
This would be a regression on mobile, the resizing style need to be applied

```suggestion
```

Reply:
Yes, I noticed this but didn't know how it ended up in here, thanks